src/lib/features/feature-toggle/types/feature-toggle-store-type.ts

```diff
@@ -103,4 +103,6 @@ export interface IFeatureToggleStore extends Store<FeatureToggle, string> {
     getFeatureTypeCounts(
         params: IFeatureProjectUserParams,
     ): Promise<IFeatureTypeCount[]>;
+
+    setCreatedByUserId(batchSize: number): Promise<void>;
```

Comment:
Should we split this into a different service? Or else we should have a task to clean it up. I'm worried about the noise we're adding into feature-toggle-store which is now +700 lines

Reply:
As discussed I moved the table reference constants into the method so it's self contained